app/assets/stylesheets/layout_components/notification.scss

```diff
@@ -111,3 +111,102 @@
     }
   }
 }
+/* Notification Centre Styling starts here */
+.right-sidebar{
+  z-index: 9999;
+  width: 350px;
+  transform: translate(350px, 0px);
+
+  &__bg{
+    @extend .right-sidebar;
+    background-color: $ash;
+  }
+
+  &__header-bg{
+    background-color: $white;
+    border-bottom: 2px solid $stone;
+    border-left: 2px solid $stone;
+    padding-bottom: $base-padding / 2;
+  }
+
+  &__notice{
+    margin-top: $base-margin;
+
+    &--active{
+      padding-top: $base-padding;
+      background-color: $sky;
+      border-left: 5px solid $cerulean;
+      border-right: 1px solid $cerulean;
+      border-top: 1px solid $cerulean;
+      border-bottom: 1px solid $cerulean;
+    }
+
+    &--inactive{
+      padding-top: $base-padding / 2;
+      background-color: $white;
+      border-left: 5px solid $smoke;
+      border-right: 1px solid $smoke;
+      border-top: 1px solid $smoke;
+      border-bottom: 1px solid $smoke;
+    }
+  }
+
+  &__notice-icon{
+    border-radius: 100%;
+    width: 35px;
+    height: 35px;
+    font-size: $medium-font-size;
+    text-align: center;
+    position: relative;
+    top: 4px;
+
+    &--red{
+      @extend .right-sidebar__notice-icon;
+      color: $white;
+      background-color: $brand-red;
+      float: left;
+    }
+    &--orange{
+      @extend .right-sidebar__notice-icon;
+      color: $white;
+      background-color: $orange;
+      float: left;
+    }
+  }
+}
+
+.notification{
+    display: inline-block;
+    position: relative;
+
+  @include breakpoint (large){
+    &__badge{
+      @include badge;
+      background: $brand-red;
+      color: $white;
+      line-height: 1.5;
+      font-weight: bold;
+      position: absolute;
+      top: 13px;
+      right: 381px;
+    }
+  }
+  @include breakpoint (medium down){
+    &__icon{
+      @include vertical-center;
+      position: absolute;
+      right: 15px;
+    }
+
+    &__badge{
+      @include badge;
+      background: $brand-red;
+      color: $white;
+      line-height: 1.5;
+      font-weight: bold;
+      position: absolute;
```

Comment:
the only things that's really different between the large and medium down screens is the positioning, so you could drop the media query for large (just make it the default), and then just have the top and right positioning in your media query - to avoid duplicating styles. so:

&__badge{
  @include badge;
  background: $brand-red;
  color: $white;
  line-height: 1.5;
  font-weight: bold;
  position: absolute;
  top: 13px;
  right: 381px;
}

@include breakpoint (medium down){
  &__badge{
    top: 13px;
    right: 5px;
  }
}